ember-cli-build.js

```diff
@@ -1,8 +1,12 @@
 /* global require, module */
-var EmberApp = require('ember-cli/lib/broccoli/ember-addon');
+const EmberApp = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  var app = new EmberApp(defaults, {});
+  const app = new EmberApp(defaults, {
+    'ember-cli-babel': {
+      includePolyfill: true
```

Comment:
Is the polyfill actually required?

Reply:
I'm pretty sure it was needed somewhere down the line - I'll try to get rid of it for this PR, it was a copy and paste thing.

Reply:
@calvinlough Ah, I remember now, if I remove it, the async await in the tests and in ember-try won't work. I looked at the ember-addon blueprints and nothing seems to suggest that this needed though - Are you aware of a better way of fixing this?

Reply:
That's unusual, I'm not sure why that's happening. It seems like this change would force the babel polyfill on anyone using the addon, which is a no-go. Do you see that error if you just run `yarn test` locally or is it a GH actions issue?

Reply:
This might be an alternative that's better suited for an addon? 

https://github.com/machty/ember-maybe-import-regenerator

Reply:
Sounds good